lmdeploy/serve/async_engine.py

```diff
@@ -235,10 +235,7 @@ def _build_turbomind(
             backend_config.session_len = self.chat_template.session_len
         from lmdeploy import turbomind as tm
         self.engine = tm.TurboMind.from_pretrained(
-            model_path,
-            engine_config=backend_config,
-            chat_template_config=chat_template_config,
```

Comment:
may remove the `chat_template_config` argument of `_build_turbomind` function as well.

Reply:
may remove the argument when calling the `__build_turbomind`